### DISCUSSION TITLE
How to add title to send_document ?

### DISCUSSION THREAD

**Discussion by terrarishabh** [filipporomani/whatsapp#50] [Q&A] [answered]
I checked the documentation but couldn't find any way to change the title/name of the file in `mesenger.send_document()`.
Could use some help with this.

**Comment by filipporomani** [comment]
Hello! You can't actually change the file name using the send_document method directly.
You would need to download the file to your local machine from the external URL, rename it, upload it on some cloud (or even make it available online from your pc) and then send the renamed file URL to the user. However I don't know how convenient this is.

**Comment by filipporomani** [comment, marked as answer]
You could upload the document to facebook using the app.upload_media() method, which gives you the file id.
Then you can do send_document(document_id, link=False). I haven't tested this, but it's the only possible solution that pops in my mind.

**Comment by terrarishabh** [comment]
Thanks for answering.
